packages/cli-build/src/commands/build/wait.js

```diff
@@ -46,8 +46,8 @@ export class Wait extends Command {
   };
 
   static examples = [
-    '$ percy build:wait --build 123',
-    '$ percy build:wait --project test --commit HEAD'
+    '$ percy build:wait --build 2222222',
+    '$ percy build:wait --project org-slug/test --commit HEAD'
```

Comment:
Should we do `org/project` or `org-slug/project-slug`?

Reply:
I think so 💯

Reply:
```suggestion
    '$ percy build:wait --project org/project --commit HEAD'
```

Reply:
Just committed it from the UI since you agree. 😆